runtime/doc/news.txt

```diff
@@ -140,6 +140,8 @@ The following changes may require adaptations in user config or plugins.
 • |nvim_open_win()| now blocks all autocommands when `noautocmd` is set,
   rather than just those from setting the `buffer` to display in the window.
 
+• Removed the |gs| normal command.
```

Comment:
"normal command" is a pedantic distinction that I don't think is useful to make to users. If the core maintainers don't know wtf "normal command" is then neither will users. We don't need to overshare of how everything was implemented internally.